api/python/notebooks/comp_bio_census_info.ipynb

```diff
@@ -1103,7 +1103,10 @@
     }
    ],
    "source": [
-    "human_cell_types = human_cell_types.query(\"is_primary_data == True\")\n",
+    "human_cell_types = census[\"census_data\"][\"homo_sapiens\"].obs.read_as_pandas_all(\n",
+    "    column_names=[\"cell_type\"], value_filter=\"is_primary_data == True\"\n",
```

Comment:
Perhaps for the future, but we need to create a notebook that provides a quick tutorial on the `value_filter` syntax.

it is quite simple, and will eventually be covered by the SOMA docs.  But as an interim, perhaps in our notebooks?

Reply:
Good idea I'll try to make one that shows the ways we can "easily" access `SOMADataFrame` for pandas operations.

Reply:
You can find the full syntax with:
```
import tiledb
help(tiledb.QueryCondition)
```